src/core/term.ml

```diff
@@ -61,7 +61,8 @@ type term =
   (** Pattern variable application (only used in rewriting rules LHS). *)
   | TEnv of term_env * term array
   (** Term environment (only used in rewriting rules RHS). *)
-  | Wild (** Wildcard (only used for surface matching, never in a LHS). *)
+  | Wild
+  | Plac of bool * string option
```

Comment:
Meaning of the arguments?

Reply:
I moved the comments to the interface file. Regarding the position of the comments, we must choose whether we duplicate them (and increase the risk of forgetting to update one of the copies), or we write them only in the interface files.

Reply:
I know that this is some extra work but I find really convenient to have the comments in the implementation files too, to avoid switching between the two files. Currently, the interface files are mainly here to define what is exported.